setup.md

```diff
@@ -24,78 +24,144 @@ title: Setup
 
 
 
-> ## Software
-> [Python](http://python.org) is a popular language for
-> scientific computing, and great for general-purpose programming as
-> well.  Installing all of its scientific packages individually can be
-> a bit difficult, so we recommend an all-in-one installer.
+> ## Installing Python using Anaconda
+> [Python][python] is a popular language for scientific computing, and great for
+> general-purpose programming as well. Installing all of its scientific packages
+> individually can be a bit difficult, however, so we recommend the all-in-one
+> installer [Anaconda][anaconda].
 >
-> For this workshop we use Python version 3.x.
->
-> ### Required Python Packages for this workshop
->
-> * [Pandas](http://pandas.pydata.org/)
-> * [Jupyter notebook](http://jupyter.org/)
-> * [Numpy](http://www.numpy.org/)
-> * [Matplotlib](http://matplotlib.org/)
+> Regardless of how you choose to install it, please make sure you install Python
+> version 3.x (e.g., 3.6 is fine). Also, please set up your Python environment at 
+> least a day in advance of the workshop.  If you encounter problems with the 
+> installation procedure, ask your workshop organizers via e-mail for assistance so
+> you are ready to go as soon as the workshop begins.
```

Comment:
I agree it is useful to remind participants to prepare a bit in advance. But currently it's a bit hidden. Do you have any suggestions how we could make it more visible?

Reply:
I am not sure where a good place for this would be. I tend to highlight this in an email I send out to participants before a workshop.

Reply:
We send similar reminders/requests to workshop participants but I'm on the fence whether this should be part of the lesson -- when participants install software seems to be more appropriate for a workshop page where all event-related questions are handled.

Reply:
Happy for this to be removed then. Though stating that python 3.x is required would be good I think.

Reply:
Yes, let's do that. And you can submit a PR to https://github.com/carpentries/workshop-template with this reminder, if it doesn't have one already.

Reply:
Ok will try and remember to do that.